packages/lexicon/src/types.ts

```diff
@@ -415,16 +415,9 @@ export function isDiscriminatedObject(
   return discriminatedObject.safeParse(value).success
 }
 
-export class LexiconDocMalformedError extends Error {
```

Comment:
Technically removing this would necessitate going to 0.3.0 on the off-chance that someone imports it. Not sure if we care about this at this point.